### PR TITLE
solved the task

### PR DESCRIPTION
**Pull request by Cerne13**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+        "status": "failed" if errors else "passed"
+    }
 
 
 def format_linter_report(linter_report: dict) -> list:
-    # write your code here
-    pass
+    return [
+        format_single_linter_file(path, error)
+        for path, error in linter_report.items()
```
Comment (MasterpieceElbow) [inline app/main.py:22]: `error` means a single object. But linter_report values are lists.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (Cerne13) [reply app/main.py]: Thank you!
Yes, it's a silly mistake. Of course we pass a list of errors to the function.

**Comment by Cerne13** [comment]
Hi! Could you re-review my task please? 
I've changed the variable name to 'errors' since it's a list and pushed a new commit.
Thank you.

**Review by Nattalli — APPROVED** [review]
👍🏻

**Pull request closed without merging**